CUDA median_filter.cu: redundant #include precomp.hpp

##### System information (version)
<!-- Example
- OpenCV => 3.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2015
-->

- OpenCV => 3.2 20170512
- Operating System / Platform => Windows 10 64 bit
- Compiler => VS2017 (VS2015 for nvcc)

[+ Forum(VS2013)](http://answers.opencv.org/question/122286/vs-2013-cannot-open-include-file-tbbtbbh-no-such-file-or-directory/)

##### Detailed description

[/modules/cudafilters/src/cuda/median_filter.cu](/opencv/opencv/tree/master/modules/cudafilters/src/cuda/median_filter.cu) includes `precomp.hpp` which includes `opencv2/core/private.hpp` which includes `tbb/tbb.h`(when `WITH_TBB` is set), which is redundant and seems unclear why precompiled header is included in the first place.

And since TBB is not included in CUDA build scripts, when trying to generate `cuda_compile_generated_median_filter.cu.obj` it produces following error: `fatal error C1083: Cannot open include file: 'tbb/tbb.h': No such file or directory`

##### Steps to reproduce
Build opencv `WITH_CUDA` and `WITH_TBB`.

--- a/modules/cudafilters/src/cuda/median_filter.cu
+++ b/modules/cudafilters/src/cuda/median_filter.cu
@@ -42,11 +42,6 @@
 
 #if !defined CUDA_DISABLER
 
-#include "precomp.hpp"
-
-using namespace cv;
-using namespace cv::cuda;
-
 #include "opencv2/core/cuda/common.hpp"
 #include "opencv2/core/cuda/vec_traits.hpp"
 #include "opencv2/core/cuda/vec_math.hpp"

